pgx/src/prelude.rs

```diff
@@ -47,3 +47,6 @@ pub use crate::pg_sys::{
     check_for_interrupts, debug1, debug2, debug3, debug4, debug5, ereport, error, function_name,
     info, log, notice, warning, FATAL, PANIC,
 };
+
+// Sub-transactions
+pub use crate::subxact::*;
```

Comment:
Probably better to limit this to just the module:  `pub use crate::subxact;`.

I think we're trying to be a little better about globbing everything into the current namespace.

Reply:
I am not sure this is the best. At the very least, I think we should consider `pub use crate::subxact::{self, SubTransactionExt}` for the sub-transaction functionality to be easily available.